pkg/ee/resource-quota/handler.go

```diff
@@ -207,9 +217,26 @@ func ListResourceQuotas(ctx context.Context, request interface{}, provider provi
 		return nil, err
 	}
 
+	// Fetching projects to get their human-readable names.
+	projectMap := make(map[string]*kubermaticv1.Project)
```

Comment:
to make it a bit lighter, could you make a map[string]string where the value is the human readable name?